### PR TITLE
[WIP] Automatic i18n Translations with Huggingface 🤗 📖 

### PR DESCRIPTION
**Pull request by martinb-ai**
The goal is to get more worldwide traction with many users from all across the globe. Let's get the ball rolling on this initiative. Always easier to add/edit files once they are already made.

Supports the following language translation from english:
- Spanish
- French
- Italian
- Portuguese (Brazilian)
- Mandarin
- Japanese
- Russian
- Arabic
- Hindi
- Indonesian

Next steps:
[ ] Build an automatic checker for new entries for new auto translation
[ ] Double check all translations with respective experts in languages

Note: The nature of this problem is challenging due to some commands being a single word or certain words having no direct translation without prior context - this is why each file will need a 30 min review from someone who is an expert in the language. You will see some files contain pure English - this is where the transformer model did not generate useful output. These all need to be translated into their respective language.

### CODE REVIEW

**Comment by jose-donato** [comment]
Wow this is crazy. 

Just a note - the pt one is not pt is pt-br

**Comment by jose-donato** [comment]
> > Wow this is crazy.
> > Just a note - the pt one is not pt is pt-br
> 
> @jose-donato Oh so this is Brazilian Portuguese?

The translation you have in the pr yes

File: openbb_terminal/miscellaneous/i18n/ru.yml [hunk 1]
```diff
@@ -0,0 +1,1153 @@
+ru:
+  _configure_: Настроить собственный терминал
```
Comment (BychkovRY) [inline openbb_terminal/miscellaneous/i18n/ru.yml:2]: _configure_: Настройте свой собственный терминал

File: openbb_terminal/miscellaneous/i18n/ru.yml [hunk 2]
```diff
@@ -0,0 +1,1153 @@
...
+  _home_: Информация, руководства и поддержка терминала OpenBB
```
Comment (BychkovRY) [inline openbb_terminal/miscellaneous/i18n/ru.yml:3]: _home_: Информация, руководства и поддержка для OpenBB Terminal

**Comment by samuelstroschein** [comment]
We'd like to chip in and provide OpenBB with a UI for contributors to add translations with [inlang/inlang](https://github.com/inlang/inlang). See as an example https://inlang.com/editor/github.com/inlang/example.

The only requirement is a [yaml plugin](https://inlang.com/documentation/plugins). We can build one.

**Comment by jmaslek** [comment]
> We'd like to chip in and provide OpenBB with a UI for contributors to add translations with [inlang/inlang](https://github.com/inlang/inlang). See as an example https://inlang.com/editor/github.com/inlang/example.
> 
> The only requirement is a [yaml plugin](https://inlang.com/documentation/plugins). We can build one.

Hey!  That sounds awesome!  What steps would be needed?

**Comment by samuelstroschein** [comment]
@jmaslek From your side, nothing! But wups. I just saw that your repo is 1GB large! 

We need to modify our git implementation to make inlang x OpenBB work. That's part of our [the next git](https://inlang.com/documentation/the-next-git) thesis though and therefore exactly what we want to do. 

I started a project https://github.com/orgs/inlang/projects/9. Can't give a concrete timeline yet.

**Comment by jmaslek** [comment]
> @jmaslek From your side, nothing! But wups. I just saw that your repo is 1GB large!
> 
> We need to modify our git implementation to make inlang x OpenBB work. That's part of our [the next git](https://inlang.com/documentation/the-next-git) thesis though and therefore exactly what we want to do.
> 
> I started a project https://github.com/orgs/inlang/projects/9. Can't give a concrete timeline yet.

No worries!  We look forward to it!

**Review by piiq — CHANGES_REQUESTED** [review]
- I would remove Hindi because India has a lot more languages than just Hindi and it's either supporting more of them or sticking with a language that is still the most dominant in India (english that is)
- I would remove Arabic because it's a right-to-left written language and we have not looked into support for RTL

P.S. @soggyomelette do you speak Arabic?

**Pull request closed without merging**